### DISCUSSION TITLE
Using the with-tailwindcss example, during development Tailwind styling works doesn't work in prod builds

### DISCUSSION THREAD

**Discussion by arn4v** [vercel/next.js#17623] [Help] [answered]
I just deployed my nextjs project that I created using create-next-app --example with-tailwindcss but in the production build tailwind doesn't work. How do I go about fixing this?

**Comment by leerob** [comment, marked as answer]
I was not able to reproduce this. If you run `yarn build && yarn start` locally, does it work? That's creating a production build, which should be the same as when it's deployed.

**Reply by arn4v** [reply]
Hi Lee, sorry for jumping to the gun it must've been the changes that I made that caused that to happen, haven't been able to reproduce it on a fresh scaffold.
